src/Aquifer.API/Endpoints/Resources/Content/Publish/Endpoint.cs

```diff
@@ -86,10 +94,35 @@ await Helpers.CreateNewDraft(dbContext,
 
                 await historyService.AddStatusHistoryAsync(mostRecentContentVersion, ResourceContentStatus.Complete, user.Id, ct);
             }
+            
+            var machineTranslation = await dbContext
+                .ResourceContentVersionMachineTranslations
+                .AsTracking()
+                .FirstOrDefaultAsync(x => x.Id == mostRecentContentVersion.Id, ct);
+
+            if (machineTranslation is not null)
+            {
+                similarityScoreMessages.Add(
+                    new ScoreResourceContentVersionSimilarityMessage(
+                        machineTranslation.Id,
+                        mostRecentContentVersion.Id,
+                        ResourceContentVersionSimilarityComparisonTypes.MachineTranslationToResourceContentVersion)
+                );
+            }
+            else
+            {
+                logger.LogInformation($"No machine translation found for published content version {mostRecentContentVersion.Id}.");
+            }
         }
 
         await dbContext.SaveChangesAsync(ct);
 
+        foreach (var similarityScoreMessage in similarityScoreMessages)
+        {
+            await scoreResourceContentVersionSimilarityMessagePublisher
+                    .PublishScoreResourceContentVersionSimilarityMessageAsync(similarityScoreMessage, ct);
```

Comment:
At this point we should probably use `CancellationToken.None` instead of passing through the `ct` because we need to publish these messages even if cancellation is requested due to having already saved to the DB.